runtime/lua/vim/treesitter.lua

```diff
@@ -199,7 +201,7 @@ end
 ---@param row number Position row
 ---@param col number Position column
 ---
----@returns (table) Table of captures
+---@return table Captures
```

Comment:
That's a table of tables whose keys are:
- `capture` : the capture name
- `priority` : the capture priority

It is worth noting that this is really tied to highlight queries.

Reply:
Agreed, allowing different captures is a feature request that should be considered for 0.9.